proxy/src/com/aerospike/client/proxy/auth/AuthTokenManager.java

```diff
@@ -328,5 +399,9 @@ public AccessToken(long expiry, long ttl, String token) {
 			this.ttl = ttl;
 			this.token = token;
 		}
+
+		public boolean hasExpired() {
+			return System.currentTimeMillis() > expiry;
```

Comment:
The client uses System.nanoTime() for timeout calculations in all other places.  It would help to be consistent to use that here as well.

Reply:
The auth token ttl returned by the proxy server is in milliseconds, hence using the same here for consistency.